assets/extensions/main.js

```diff
@@ -30,7 +30,19 @@ const Main = () => {
 			.catch( () => setExtensions( [] ) );
 	}, [] );
 
-	if ( false === extensions ) {
+	const [ layout, setLayout ] = useState( false );
+
+	useEffect( () => {
+		apiFetch( {
```

Comment:
This could have a refactor when it gets merged and https://github.com/Automattic/sensei/pull/4205 too.